lib/shopify_transporter/exporters/magento/sql.rb

```diff
@@ -28,8 +28,9 @@ def connect
             host: @host,
             port: @port,
             database: @database
-          )
-          yield(@connection)
+          ) do |db|
+            yield db
+          end
```

Comment:
We're not reusing `@connection` anywhere, I suggest removing the assignment to the instance variable.

Maybe rename the method to `with_connection` to match how we yield objects like this in other projects?

Reply:
The underlying sequel lib yields a connection in `Sequel#connect`, which was the rationale behind keeping it consistent in the wrapper. I don't have a preference between `with_` `connection`, though